extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -158,3 +156,31 @@ amp-ad[sticky='bottom-right'] {
   bottom: 0;
   right: 0;
 }
+
+amp-ad[sticky='left'],
+amp-ad[sticky='right'] {
+  background: #fff;
+}
+
+amp-ad[sticky='left'] iframe,
+amp-ad[sticky='right'] iframe {
+  max-width: 100px !important;
```

Comment:
I think max-width should be 120px as 120x600 is pretty standard size that news publishers use.

Reply:
That is fair. Adjusted.